packages/admin/src/pages/scheme/[schemeId]/[exportId]/index.page.tsx

```diff
@@ -0,0 +1,232 @@
+
+    if (unavailableSubmissionsTotalCount > 0) {
+      const failedExportDetails = await getExportDetails(
+        exportId,
+        true,
+        pagination,
+        sessionCookie
+      );
+      unavailableSubmissions = failedExportDetails.exportedSubmissions;
+    }
+  } catch (err) {
+    return generateErrorPageRedirect(
+      'Something went wrong while trying to view submission applications.',
+      '/dashboard'
+    );
+  }
+
+  if (
+    availableSubmissionsTotalCount == 0 &&
+    unavailableSubmissionsTotalCount == 0
+  )
+    return generateErrorPageRedirect(
+      'There are no submissions available to view or download.',
+      '/dashboard'
+    );
+
+  return {
+    props: {
+      formAction: process.env.SUB_PATH + resolvedUrl,
+      csrfToken: res.getHeader('x-csrf-token') as string,
+      individualApplicationsHref: `/scheme/${schemeId}/${exportId}/download-individual`,
+      superZipLocation: superZipLocation,
+      schemeName: grantScheme.name,
+      availableSubmissionsTotalCount: availableSubmissionsTotalCount,
+      unavailableSubmissionsTotalCount: unavailableSubmissionsTotalCount,
+      unavailableSubmissions: unavailableSubmissions,
+      schemeId: schemeId,
+      exportId: exportId,
+    },
+  };
+};
+
+export const CompletedSubmissions = ({
+  formAction,
+  csrfToken,
+  schemeName,
+  individualApplicationsHref,
+  availableSubmissionsTotalCount,
+  unavailableSubmissionsTotalCount,
+  unavailableSubmissions,
+  superZipLocation,
+  schemeId,
+  exportId,
+}: InferGetServerSidePropsType<typeof getServerSideProps>) => {
+  const tableHeadColumns = [
+    {
+      name: <>{true && 'Organisation'}</>,
+      width: 'three-quarters',
+    },
+    {
+      name: 'Action',
+      isVisuallyHidden: true,
+    },
+  ] as TheadColumn[];
+
+  const tableRows = unavailableSubmissions.map((submission) => {
+    return {
+      cells: [
+        {
+          content: <>{true && submission.name}</>,
+        },
+        {
+          content: (
+            <div className="govuk-!-text-align-right">
+              <CustomLink
+                href={`/scheme/${schemeId}/${exportId}/view/${submission.submissionId}`}
+                ariaLabel={`View submission "${submission.name}"`}
+              >
+                View
+              </CustomLink>
+            </div>
+          ),
+        },
+      ],
+    };
+  });
+
+  return (
+    <>
+      <Meta title={`Download applications - Manage a grant`} />
+      <div className="govuk-grid-row govuk-!-padding-top-7 govuk-!-margin-0 govuk-!-margin-bottom-6">
+        <div className="govuk-grid-column-full-width">
+          <div className="govuk-!-width-two-thirds">
+            {unavailableSubmissionsTotalCount > 0 && (
+              <DownloadMessage count={unavailableSubmissionsTotalCount} />
+            )}
+
+            <h1 className="govuk-heading-l">{schemeName}</h1>
+
+            <h2 className="govuk-heading-m">Applications submitted</h2>
```

Comment:
on the figma designs this says "Applications available to download" rather than "Applications submitted". Not sure if there's been a change to designs

Reply:
Will be in next push!